api/src/main/java/com/codeforcommunity/dto/UserReturn.java

```diff
@@ -0,0 +1,56 @@
+package com.codeforcommunity.dto;
+
+public class UserReturn {
+  // doesnt make sense that we would ever set these values after construction.
+  // favor immutibility.
+  public final int id;
+  public final String email;
+  public final String firstName;
+  public final String lastName;
+  public final String year;
+  public final String major;
+  public final int privilegeLevel;
+
+  public UserReturn(int id, String email, String firstName, String lastName, String year, String major,
+      int privilegeLevel) {
+    this.id = id;
+    this.email = email;
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.year = year;
+    this.major = major;
+    this.privilegeLevel = privilegeLevel;
+  }
+
+  public String toString() {
```

Comment:
Add an `@Override` tag to make it clear you're overloading this method. A little nit-picky but this method should probably be at the bottom of the file.